roles/ceph-common/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ ceph:
 
   openstack_keys:
     - { name: client.glance, value: "mon 'allow r' osd 'allow class-read object_prefix rbd_children, allow rwx pool=default'" }
-    - { name: client.cinder, value: "mon 'allow r' osd 'allow class-read object_prefix rbd_children, allow rwx pool=default'" }
+    - { name: client.cinder, value: "mon 'allow r' osd 'allow class-read object_prefix rbd_children, allow rwx pool=default, allow rwx pool=vms'" }
```

Comment:
Should mention that the pool name listed here needs to match the pool name used below for `vm_pool_name`.